infra/license/license-checker.ts

```diff
@@ -31,6 +31,129 @@ var args = process.argv.slice(2)
 const inputPathOfLicenseList = args[0];
 const verificationTag = args[1];
 
+const VerificationResultType = {
```

Comment:
(sorry but, I'm not good at so that I'm asking.)

Q. Is it a kind of global const map? like in c++
```cpp
const std::unordered_map<std::string, std::string> VerficationResultType{
  "pass": "No license issue found', ...
};
```

Reply:
AFAIK, this is not const map but an class in C++, e.g.,

```
class VerificationResultType {
  public: 
    const std::string pass = "No license issue found";
};
```

Reply:
Aha!

```c++
class VerificationResultType
{
  public:
    const std::string pass() { return "No license issue found"; }
};
```

If so, this could be a kind of enum or a global const singleton class.

Reply:
It's something like

```cpp
enum VerificationResultTypeKeys {
  pass,
  warnConditionalLicense,
  ...
};

const std::unordered_map<VerificationResultTypeKeys, std::string> VerificationResultType{
  VerificationResultTypeKeys.pass: "No license issue found",
  ...
};
```

Please refer to https://github.com/Samsung/ONE-vscode/pull/393#discussion_r832777862 for little bit more detailed explanation! :D